portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -0,0 +1,73 @@
+
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.Query.FilterPredicate;
+import com.google.appengine.api.datastore.Query.FilterOperator;
+import com.google.appengine.api.datastore.KeyFactory;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.gson.Gson;
+import com.google.sps.data.Fields;
+import com.google.sps.data.Video;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+// Servlet for testing queries.
+@WebServlet("/data")
+public class DataServlet extends HttpServlet {
+
+    @Override
+    public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException{
+        String id = request.getParameter(Fields.VIDEO_ID);
+        if(id == null){
+            response.getWriter().println("AAAAAAAAAAAAAAAAAAAAAA");
+        }
+        Key idAsKey = KeyFactory.createKey("Video", id);
+        DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
NIT: You might be able to make this a static variable and avoid creating a new service for every request.